tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/android_anr.pysnap

```diff
@@ -808,14 +808,14 @@ system:
               "zygoteinit.java"
             function*
               "run"
-          frame
+          frame (ignored by stack trace rule (category:threadbase -group v-group))
```

Comment:
I don't quite understand why the frame did not have a hint to begin with, while all the frames before and after have it.